dominos/src/main/java/me/fontys/semester4/dominos/pizza/and/ingredient/PizzaAndIngredientRecord.java

```diff
@@ -0,0 +1,125 @@
+package me.fontys.semester4.dominos.pizza.and.ingredient;
+
+import com.opencsv.bean.CsvBindByPosition;
+
+public class PizzaAndIngredientRecord {
```

Comment:
Ik stel voor om wel alles conventioneel Engels te houden? Dat maakt het ook leesbaar voor o.a. Gleb :)

Reply:
Can't find the reference (outdated). Did you mean the original filename? I could change it.